src/fits.jl

```diff
@@ -167,3 +167,17 @@ function subtract_dark!(frame::AbstractArray, dark_frame::ImageHDU; dark_exposur
 end
 
 subtract_dark!(frame::AbstractArray, dark_frame::String; hdu = 1, kwargs...) = subtract_dark!(frame, FITS(dark_frame)[hdu]; kwargs...)
+
+
+"""
+    combine(frames::Vararg{<:FITSIO.ImageHDU}; kwargs...)
+    combine(filename::Vararg{<:String, N}; hdu = ntuple(one, N),  kwargs...)
```

Comment:
docstrings like this are hard to read, we don't need to copy the method signature exactly (it's easier at that point for users to just read the source code!) Why not something like this-
```suggestion
	combine(frames...; hdu=1, kwargs...)
	
Load multiple FITS files or HDUs before combining. If loading from filenames, the HDU number can be specified with `hdu` as either an integer or a tuple corresponding to each file.
```